src/main/java/Euler.java

```diff
@@ -0,0 +1,14 @@
+import java.util.stream.IntStream;
+
+public class Euler {
+
+    final static private int START_NUMBER = 1;
```

Comment:
To `private` raczej na początek. Wszyscy się RACZEJ trzymają konwencji `private static final ...`